neural_compressor/adaptor/tf_utils/quantize_graph/qdq/fuse_qdq_conv.py

```diff
@@ -1795,6 +1795,10 @@ def _is_match_conv(self, patterns, qdq_inserted=False):
                 if cur_node.name != self.start_node_name:
                     continue
 
+                # Disable inserting QDQ for DepthwiseConv2dNative since its poor perf.
+                if cur_node.op == "DepthwiseConv2dNative" and self.itex_mode:
```

Comment:
shouldn't it to be fixed by ITEX? why we disable it?

Reply:
The efficiency of the INT8 'DepthwiseConv2dNative' is not good, oneDNN team is checking. oneDNN graph team asked us to disable them temporarily to reach the OOB performance goal.